### DISCUSSION TITLE
Add ability to adjust hitsound volume and other sfx volume seperately

### DISCUSSION THREAD

**Discussion by iceCreamMint** [ppy/osu#29156] [Ideas] [answered]
On mobile i have high audio lag, so i have to turn hitsounds off. However, since in lazer hitsound volumes and other sfx volumes seem to be controlled by the same slider, turning off hitsounds also turns off all the main menu sound effects. For keysounded maps another feature to consider is the abilty to play beatmap hitsounds regardless of whether the note is hit.

**Comment by Stoppedpuma** [comment, marked as answer]
Duplicate of #25068, a PR can be found at #28733 for this feature as well.
